subgraphs/uniswap-forks/protocols/spiritswap/config/networks/fantom/fantom.ts

```diff
@@ -51,7 +51,7 @@ export class SpiritSwapFantomConfigurations implements Configurations {
     );
   }
   getTradeFee(): BigDecimal {
-    return BigDecimal.fromString("0.3");
+    return BigDecimal.fromString("0.18");
```

Comment:
do we need to redeploy this change?

Reply:
No this was my bad. I meant to revert this change. When I was sifting through the documents, I thought it might be wrong. I need to look into this more tomorrow. In the docs it talked about this change being made with SpiritswapV2.

Reply:
SpritiswapV2 has been out for about a month now with a different set of contracts. So this change is irrelevant - though, we might want to consider indexing this one!